assets/scss/modules/_breadcrumb.scss

```diff
@@ -33,12 +33,13 @@
   li.active {
     color: $dp-color-lightgrey;
 
-    :before {
+    &:before {
       margin-right: 5px;
     }
 
     a {
       padding-left: 5px;
+      color: $dp-color-lightgrey;
```

Comment:
why do you need a special style for the `a` elements inside active breadcrumb item?